easybuild/framework/easyblock.py

```diff
@@ -1546,6 +1546,10 @@ def sanity_check_step(self, custom_paths=None, custom_commands=None, extension=F
         - if *any* of the files/subdirectories in the installation directory listed
           in sanity_check_paths are non-existent (or empty), the sanity check fails
         """
+        # constants for library checking (TODO: should probably be defined somewhere else/configurab;e)
+        LIB_DIRS = ['lib', 'lib64']
+        LIB_SUFFIXES = {'static': ['.a'], 'shared': ['.so', '.dyn']}
```

Comment:
constant should be defined at the top of this file, below imports

rather than hardcoding `.so` or `.dyn`, use `get_shared_lib_ext()` from `easybuild.tools.systemtools`